src/main/java/org/broadinstitute/hellbender/engine/SkeletonVariant.java

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.hellbender.engine;
+
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+/**
+ * SkeletonVariant is a minimal implementation of the Variant interface.
+ */
+public class SkeletonVariant implements Variant {
+
+    private SimpleInterval interval;
+    private Boolean isSNP;
+    @Override
+    public int getStart() {
+        return interval.getStart();
+    }
+    @Override
+    public int getEnd() {
+        return interval.getEnd();
+    }
+    @Override
+    public boolean isSnp() {
+        return isSNP;
+    }
+    @Override
+    public boolean isIndel() {
+        return !isSNP;
+    }
+
+    public SkeletonVariant(SimpleInterval interval, Boolean isSNP) {
```

Comment:
Use consistent construction conventions between the two implementations of `Variant` -- either use a public constructor (as here), or a private constructor plus a public factory method. The first option would be more consistent with the rest of the hellbender codebase.